data/eddiejaoude.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Eddie Jaoude",
+  "url": "https://github.com/eddiejaoude",
+  "image": "https://github.com/eddiejaoude.png"
```

Comment:
There's a missing comma in this line.
We could add a github action to lint these files.

Reply:
```suggestion
  "image": "https://github.com/eddiejaoude.png",
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix